### DISCUSSION TITLE
New Metadata API inserting Title tag in the body

### DISCUSSION THREAD

**Discussion by st11x** [vercel/next.js#47409] [Help]
### Summary

Hi,

I added this code to the topmost layout.tsx file in my app folder. There are no other places where I have set the metadata object. 

```javascript
export const metadata = {
    title: "Admin Portal"
};
```

When the page is rendered, I do see the title tag in the browser, but I also see this in the source.

![image](https://user-images.githubusercontent.com/7398499/226991862-e934775e-9362-467f-ac69-575dab30e71b.png)

Is this expected?

**Comment by icyJoseph** [comment]
This does look wrong... I wonder if the suspense boundary is to blame here. Do you see any errors whiles rendering?

Could you also please try to share a bit more code, to try and reproduce this?

## Edit

I think you have found a bug. When there's a `loading.tsx` kind of file, the meta data falls to the body, which is not good...

I was able to guess this from the `<!-- $ -->`, `<!-- /$ -->` suspense boundary... 

- https://github.com/vercel/next.js/issues/46738

**Reply by st11x** [reply]
Thanks for confirming that. I also noticed that it doesn't do this when you are in production mode.
